packages/visualizations/src/components/Table/types.ts

```diff
@@ -34,22 +35,32 @@ type BaseColumn = {
     onClick?: () => void;
 };
 
-// Eliminate rawValue from props to match column options
-type ColumnOptions<T> = Omit<T, 'rawValue'>;
+export type ValueOrAccessor<T> = T | ((r: Record<string, unknown>) => T);
 
-type FormatPropsTypeMap = {
-    [DATA_FORMAT.boolean]: ColumnOptions<BooleanFormatProps>;
-    [DATA_FORMAT.date]: ColumnOptions<DateFormatProps>;
-    [DATA_FORMAT.geo]: ColumnOptions<GeoFormatProps>;
-    [DATA_FORMAT.shortText]: ColumnOptions<ShortTextFormatProps>;
-    [DATA_FORMAT.longText]: ColumnOptions<LongTextFormatProps>;
-    [DATA_FORMAT.number]: ColumnOptions<NumberFormatProps>;
-    [DATA_FORMAT.url]: ColumnOptions<URLFormatProps>;
+export type FormatPropsTypeMap = {
```

Comment:
I got an error when I try to declare a column
`options.rawValue` shouldn't be required

![image](https://github.com/user-attachments/assets/c9b79c21-cfba-4f3e-8f4a-b3209d8b1b13)
```
Type '(r: Record<string, unknown>) => { display(v: string): string; }' is not assignable to type 'ValueOrAccessor<DateFormatProps> | undefined'.
  Type '(r: Record<string, unknown>) => { display(v: string): string; }' is not assignable to type '(r: Record<string, unknown>) => DateFormatProps'.
    Property 'rawValue' is missing in type '{ display(v: string): string; }' but required in type 'DateFormatProps'
 ```

Reply:
I still have the error. Can also see when running `npm run watch`
 > Type 'Record<string, unknown>' is not assignable to type 'Record'
 
 Shouldn't Column type be generic ?